server/src/main/java/org/opensearch/index/store/remote/filecache/FileCache.java

```diff
@@ -49,8 +50,13 @@ public class FileCache implements RefCountedCache<Path, CachedIndexInput> {
 
     private final CircuitBreaker circuitBreaker;
 
-    // TODO: Convert the constant into an integer setting
-    public static final int DATA_TO_FILE_CACHE_SIZE_RATIO = 5;
+    public static final Setting<Double> DATA_TO_FILE_CACHE_SIZE_RATIO_SETTING = Setting.doubleSetting(
```

Comment:
Let's add a good comment here describing what this means and how it works. Something like:

> Defines a limit of how much total remote data can be referenced as a ratio of the size of the disk reserved for the file cache. For example, if 100GB disk space is configured for use as a file cache and the `remote_data_ratio` of 5 is defined, then a total of 500GB of remote data can be loaded as searchable snapshots. This is designed to be a safeguard to prevent oversubscribing a cluster. Specify a value of zero for no limit, which is the default for compatibility reasons.

Reply:
Added this in verbatim. Thanks for the description :)